### DISCUSSION TITLE
[通告]雅努斯区市民政府入驻绳网

### DISCUSSION THREAD

**Discussion by NE-Janus** [share121/inter-knot#251] [其他]
**亲爱的市民朋友们：**

  为加快网络改革，促进空洞开发民间化，发扬新艾利都探索精神，新艾利都雅努斯区市民政府正式入驻绳网。

  自旧都陷落后，我们与空洞的对抗从未停止，我们失去了无数宝贵的生命，付出了无数鲜血，因而得以延续文明。出于保护安全市民的原因，长久以来，我们严格把控空洞调查权利，以治安局、防卫军、HIA三位一体为主要火力手段，凭借“防守堵击，逐步蚕食，削弱空洞，利用空洞”的手段，稳定了新都的社会。但我们忽略了广大市民的强大力量与创造性，忽略了“盗洞客”与绳匠们的特殊力量，既给相关人员带来了不便，也客观上加重了空洞的影响。因此，我们郑重宣布开放绳网与官方渠道的正确合理合作，并开辟此账号作为公告媒体。相关合作法案也在商议中。

**[[新艾利都雅努斯区市民政府]宣**

**Comment by wushanef** [comment]
也就是说咱可以光明正大的接委托了？

**Comment by Phaeton9528** [comment]
你信这个还是信我是虚狩
